### ISSUE TITLE
[IMPROVEMENT] Extend the number of information on the Widget 

### ISSUE THREAD

**Issue by Moostek** [breezy-weather/breezy-weather#795]
### Describe your idea

Hello, I would like to suggest to **extend the number of information on the Widget with these recommendations**: 

1.) Extend the weather forecast from 5-day to 7-day in the widget. Extend the same items, from 5 to 7, to hour forecast. 

2.) The name of the city or actual location on the widget make with a bigger font and on 1st place (left upper corner). 

3.) Add the selectable measurements as feel temperature, rainfall, wind, humidity and other ones and place them on the right upper area of the widget. It will be great if these measurements will be represented with a small live icons (or possibly, there can be second line). Now there is just Quality of air, which I am not looking for (it is not selectable), and it is too long and big too. I suggest to create set of icons for Quality of air, so it will be easy to put it on right upper side together with other selectable measurements - with some limit, of course, like maximum of 3, for example. 

4.) Add the refresh button on the widget to refresh the weather measurements. This button shuld be placed on the end of three small icons (see the point 3.). 

5.) In general I suggest to make left upper side a bit smaller (esp. incl. the weather icon and the value in °C temperature) to allow to display small icons of other measurement (see the point 3) in right upper corner. 

Thank you. 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] My idea is not related to a question covered in the [Frequently Asked Questions](https://github.com/breezy-weather/breezy-weather/blob/main/HELP.md) document, or a feature described in the [Homepage](https://github.com/breezy-weather/breezy-weather/blob/main/docs/HOMEPAGE.md) document.
- [X] I have updated the app to **[version 5.1.3 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
Please make a mockup

**Comment by papjul** [comment]
Alright, so you want to create a new style for the notification-widget, right?

Can you show a mockup of your idea so it's easier to understand what you have in mind? :)

**Comment by Moostek** [comment]
Yes, we can called it "new style for Notification Widget with extended information", as described in points 1-5 above. 

In general I suggest to add another style for Notification Widget called somehow "Daily forecast + actual weather". And as for this "actual weather" part: there should be 3 selectable items to be shown (like feel temperature, rain, humidity, pressure, UV index etc.). 

Here is a little mockup of the "Daily forecast + actual weather" style showing up the Daily weather measurements in upper part of Notification Widget ;) 

![LOS_Breezy Weather_Nw_04_new_draw](https://github.com/breezy-weather/breezy-weather/assets/59518454/d3e43a7b-7f98-4dd7-b652-b8aec6f9cbb6)

**Comment by papjul** [comment]
You have wrong proportions.

This is how much you can fit with a good design that can breathe:
![image](https://github.com/breezy-weather/breezy-weather/assets/1449159/1be388d7-93d9-40a0-80e8-e91d688c06a2)

What I suggest instead is a "Current" style like this (without the orange background of course):
![image](https://github.com/breezy-weather/breezy-weather/assets/1449159/a2c2d3e7-1d09-4cb5-b196-769ccd247d32)

Taking "Details in header" parameters in order.

Note that screenshots were taken with "normal" accessibility settings. You have take into account that some people have increased display/font size, so we can't fit too much info.
